class ValidationError extends Error {}

const handleError = (err, req, res, next) => {
    console.error(err)

    res
        .status(err instanceof ValidationError ? 400: 500)
        .render('error', {
           message: err instanceof ValidationError ? err.message : 'Przepraszamy, spr√≥buj ponownie za kilka minut.',
        });
};

module.exports = {
    handleError,
    ValidationError
}